pkg/config/errors.go

```diff
@@ -43,6 +60,15 @@ func (err *InvalidSettingError) Error() string {
 	return fmt.Sprintf("invalid setting provided to a module, cause: %v", err.cause)
 }
 
+// InvalidModuleError signifies a problem with the supplied module name.
+type InvalidModuleError struct {
+	modID string
+}
+
+func (e InvalidModuleError) Error() string {
+	return fmt.Sprintf("%s: %s", errorMessages["invalidMod"], e.modID)
```

Comment:
No, need for `errorMessages["invalidMod"]`, just use string literal here, and remove it from `errorMessages`.

Reply:
Done